src/Raven.Server/Web/Studio/LicenseHandler.cs

```diff
@@ -9,6 +10,34 @@ namespace Raven.Server.Web.Studio
 {
     public class LicenseHandler : RequestHandler
     {
+        [RavenAction("/license/eula", "GET", AuthorizationStatus.UnauthenticatedClients)]
```

Comment:
I'm not too happy about the authorization status here.
We shouldn't make it anyone. Ideally, this is something that only operator can do, especially the `accept` portion.

I know that we are doing this from the wizard, but can we check the permissions here explicitly if we aren't in the wizard?

Reply:
Done. Only Operator can accept the EULA.

Reply:
This is running as part of the wizard, right? In that case, would there be _any_ security there?
I _think_ that in the wizard, we have no auth and anyone can do anything, in which case, let's mark this endpoint as well as `ValidUser` and go forward.

Reply:
Actually this is running before the wizard, so it also covers a case when the wizard is turned off also.

Changed to ValidUser.